pkg/apiserver/proxy.go

```diff
@@ -151,6 +151,8 @@ func (r *ProxyHandler) ServeHTTP(w http.ResponseWriter, req *http.Request) {
 	}
 	httpCode = http.StatusOK
 	newReq.Header = req.Header
+	newReq.ContentLength = req.ContentLength
+	newReq.TransferEncoding = req.TransferEncoding
```

Comment:
tl;dr: we can keep this line, though it may not be useful.

After more digging, I found actually the http package is powerful enough deduce the outbound request should be "chunked" based on the fact that the content-length is left unset && there is content:
https://golang.org/src/net/http/transfer.go#L88

Well, it doesn't hurt to keep the copy in the PR, and *perhaps* it will save a broken request where "transfer-encoding:chunked" and "content-length: non-zero-value" are set at the same time.